editor/src/components/inspector/sections/style-section/container-subsection/radius-row.tsx

```diff
@@ -52,23 +52,24 @@ export const RadiusRow = React.memo(() => {
       items={contextMenuItems}
       data={null}
     >
-      <UIGridRow
-        tall
-        padded={false}
-        variant='<---1fr--->|------172px-------|'
-        style={{ marginTop: 12 }}
+      <div
```

Comment:
Should this become another variant of `UIGridRow`? Genuine (not loaded) question